### ISSUE TITLE
Support highlighting fenced code blocks with the "console" language identifier?

### ISSUE THREAD

**Issue by achingbrain** [TypeStrong/typedoc#2123]
## Search Terms

- console
- highlighting
- markdown

## Problem

Github highlights fenced code blocks with the `console` language identifier:

````
```console
$ some-command
command output 1
command output 2
command output 3
etc
```
````

Formatted:

```console
$ some-command
command output 1
command output 2
command output 3
etc
```

Typedoc prints a warning when it finds `console` blocks:

```
→ warning Unsupported highlight language "console" will not be highlighted. Run typedoc --help for a list of supported languages.
```

## Suggested Solution

It would be neat if Typedoc could support the same set of languages as GitHub.

**Comment by Gerrit0** [comment]
TypeDoc is dependent on Shiki for which languages are supported, so this really needs to be a request there

**Comment by achingbrain** [comment]
This was fixed in Shiki and it's been released.  This module depends on [0.12.x](https://github.com/TypeStrong/typedoc/blob/master/package.json#L30) but the fix is in 0.13.x.  0.14.x is also available - can the dep be updated to pull the fix in?
